src/lib/hub-connection.ts

```diff
@@ -288,4 +283,41 @@ export class HubConnection<THub> {
 		return data;
 	}
 
+	private getReconnectOnDisconnect$(): Observable<void> {
+		let retryCount = 0;
+		const timer$ = (): Observable<void> => {
```

Comment:
minor but would rename 
```suggestion
		const delay$ = (): Observable<void> => {
```